CONTRIBUTING.md

```diff
@@ -0,0 +1,58 @@
+#How to contribute
+Alfresco's maturity comes from the contributions of its community. Contributing back to Alfresco can be a bit confusing as Alfresco is a collection of separate open source projects. Each open source project officially sponsored by Alfresco has a page in this space detailing its governance policy and the method for reporting issues and submitting contributions.
+ 
+For the details on the Records Management project see[here.](https://community.alfresco.com/docs/DOC-6387-project-overview-records-management)
+
+##Getting started
+What you need to get started:
```

Comment:
I think the getting started section should include a link to https://community.alfresco.com/docs/DOC-6269-submitting-contributions#jive_content_id_Signing_a_Contribution_Agreement

Reply:
There is a link to this in the making changes section (2nd part starting "If you are Adding") but this can be moved here if you think it will work better?

Reply:
just seen the comment in that section. Ignore this

Reply:
I didn't notice it at first. I think mentioning up front that we require a contributor agreement would be good.